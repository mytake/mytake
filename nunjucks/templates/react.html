{% extends "base.html" %}

{% block endbody %}
  <!-- Dependencies -->
  <script src="https://unpkg.com/react@15/dist/react.js"></script>
  <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>

  <!-- Main -->
  <script src="./bundle-0.0.3.js"></script>
{% endblock %}
