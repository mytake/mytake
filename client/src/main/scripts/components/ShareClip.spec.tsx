import * as React from "react";
import * as renderer from "react-test-renderer";
import ShareClip from "./ShareClip";
import { videoFactLink } from "../utils/testUtils";

test("It renders", () => {
  const tree = renderer
    .create(
      <ShareClip highlightedRange={[0.1, 5]} videoIdHash={videoFactLink.hash} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
