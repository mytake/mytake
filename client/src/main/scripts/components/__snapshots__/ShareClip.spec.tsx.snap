// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders 1`] = `
<div
  className="shareclip"
>
  <h3
    className="shareclip__title"
  >
    Share your clip
  </h3>
  <div>
    <label
      className="shareclip__label"
      htmlFor="cliptitle"
    >
      Title
    </label>
    <input
      className="shareclip__input"
      id="cliptitle"
      onChange={[Function]}
      type="text"
      value=""
    />
    <button
      className="shareclip__button"
      onClick={[Function]}
    >
      Get a shareable URL
    </button>
    <button
      className="shareclip__button"
      onClick={[Function]}
    >
      Facebook
    </button>
    <button
      className="shareclip__button"
      onClick={[Function]}
    >
      Twitter
    </button>
  </div>
</div>
`;
